.gitignore

```diff
@@ -170,3 +170,6 @@ cython_debug/
 
 /.luarc.json
 _dev/
+
+# license files should not be commited to this repository
+*.lic
```

Comment:
This also exists here: https://github.com/posit-dev/posit-sdk-py/blob/56c6eed21a7d2ca3abb9d9bccb5986f097cb77a2/integration/.gitignore#L1

The integration tests expect it to exist in the integration directory.

<img width="169" alt="Screenshot 2025-02-28 at 15 36 14" src="https://github.com/user-attachments/assets/0f86b003-ab04-491d-8e3e-c8df79030677" />

Reply:
Ah, good catch. Would you rather only have it at the top level (since that syntax will catch all instances no matter where they are) or in both places? Either is fine, both places is a little bit silly, but makes the intent clear(er)?

Reply:
The top level is better since it shouldn't be checked in anywhere.

Reply:
Done, thanks!